### ISSUE TITLE
rubyonrails getting started guide NoMethodError in Articles#index

### ISSUE THREAD

**Issue by aliimirza** [rails/rails#22375]
SUBJECT:
URL: http://guides.rubyonrails.org/getting_started.html
SECTION: 5.8 Listing all articles

CHANGE REQUEST:
The code contains variable @articles which generates NoMethodError in Articles#index

class ArticlesController < ApplicationController
  def index
    @articles = Article.all
  end

Changing @articles to @article fixed the issue. Requesting you to please make this change in the guide code.

Thank you.
Br
Mirza

**Comment by kaspth** [comment]
It's common to pluralize the assigned instance variable in `index`, since it works on a collection. This the documentation looks correct to me.

Can you elaborate on how it generates a `NoMethodError`, and what method the error says is missing?

**Comment by aliimirza** [comment]
Thank you for reply. I got the point and fixed the code which is working.

Just to explain what happened, it was index method under Articles class where ```@articles``` was causing error however removing s at the end fixed the problem. The error message pointed to corresponding view where app was failing:
```
Showing /home/ubuntu/blog/app/views/articles/index.html.erb where line #9 raised:
undefined method `each' for nil:NilClass
Extracted source (around line #9):

7	</tr>
8
9	<% @article.each do |article| %>
10	<tr>
11	<td><%= article.title %></td>
12	<td><%= article.text %></td>           
Rails.root: /home/ubuntu/blog
```
I noticed that the variable name under ```view/index.html.erb``` was written as singular but under ```controller/articles.controller.rb``` it was pluralized. I changed variable's name under ```view/index.html.erb``` to ```@articles``` by following pluralizing practice. Now everything works fine. Earlier I was singularizing variable under controller as it was already wirrten as singular under view. That also fixed the problem. 

I have been usng copy/paste method to put multiple lines of code in c9 IDE to avoid typos. It is strange that variable name was mismatched.

I have revised the code on guide, it has ```@articles``` as variable name under both controller and view which is correct, I wonder if it has been recently fixed?
